index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head><title>Salmon Cookies</title>
+  <script src="cookieapp.js"></script>
```

Comment:
your index.html doesn't need to import your js file.  This will throw an error in the console.